test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -455,6 +564,18 @@ func runWorkload(b *testing.B, tc *testCase, w *workload) []DataItem {
 			})
 			nextNodeIndex += concreteOp.Count
 
+		case *createNamespacesOp:
+			nsPreparer, err := newNamespacePreparer(concreteOp, client)
+			if err != nil {
+				b.Fatalf("op %d: %v", opIndex, err)
+			}
+			if err := nsPreparer.prepare(); err != nil {
+				b.Fatalf("op %d: %v", opIndex, err)
```

Comment:
what if some namespaces were successfully created?

Reply:
not sure I get what is the concern, this is a fatal error, so the whole the test case will fail

Reply:
Is there anything else clearing the namespaces? Isn't the etcd db shared for the entire test suite?

Reply:
ah, ok, I added a call to cleanup()